docker/packager/packager

```diff
@@ -309,21 +308,20 @@ def parse_env_variables(
         result.append("MAKE_RPM=true")
         result.append("MAKE_TGZ=true")
 
-    if with_binaries == "programs":
-        result.append("BINARY_OUTPUT=programs")
-    elif with_binaries == "tests":
-        result.append("ENABLE_TESTS=1")
-        result.append("BINARY_OUTPUT=tests")
+    if with_binaries or clang_tidy:
```

Comment:
Do we need utils in EVERY deb package? https://s3.amazonaws.com/clickhouse-test-reports/51243/5f31095b99b93c74843a5229f063a76ae0088f46/clickhouse_build_check/report.html

Reply:
As I understand we will build them and export it as binary, but it may be not packed in deb at all. I do not think it is added in deb definition

Reply:
Yes, we don't add them. But build and upload still.